src/api/java/com/minecolonies/api/entity/citizen/VisibleCitizenStatus.java

```diff
@@ -0,0 +1,48 @@
+package com.minecolonies.api.entity.citizen;
+
+import com.ldtteam.structurize.util.LanguageHandler;
+import com.minecolonies.api.util.constant.Constants;
+import net.minecraft.util.ResourceLocation;
+
+/**
+ * Enum for citizen status icons, resource location and translation
+ */
+public enum VisibleCitizenStatus
```

Comment:
This is not extensible.
Please refactor this in a way that it does not require the enum to be used.

In other words the statusses that a citizen can be in, need to be extendible via the API.

Reply:
made it a simple class instead